src/AST-Core-Tests/RBScannerTest.class.st

```diff
@@ -9,7 +9,12 @@ Class {
 
 { #category : #initialize }
 RBScannerTest >> buildScannerForText: source [
-	^RBScanner on: (ReadStream on: source)
+	^ self parserClass on: (ReadStream on: source)
+]
+
+{ #category : #initialize }
+RBScannerTest >> parserClass [
```

Comment:
I think It should be named #scannerClass.

Reply:
Yes you are rigth I was distracted.